manifests.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: powerdummy-populator
+  namespace: relationships
```

Comment:
add this

annotations:
    strategy.spinnaker.io/recreate: "true"

Reply:
fixed